src/OrchardCore.Modules/OrchardCore.Google/Views/GoogleTagManagerSettings.Edit.cshtml

```diff
@@ -0,0 +1,18 @@
+@using OrchardCore.Google.TagManager.ViewModels;
+@model GoogleTagManagerSettingsViewModel;
+
+<p class="alert alert-warning">
+    @T["The current tenant will be reloaded when the settings are saved."]
```

Comment:
I don't see a reset for the shell in the driver?

It doesn't look like it would be necesary to reset, or has the filter cached the result?

Reply:
Yes I think _scriptsCache is a cached value in the GoogleTagManagerFilter. But I don't see where we reset the tenant. If we don't reset it then we don't reset it also for the GoogleAnalytics MeasurementId.

Reply:
I think the issue is that it is not cached and that it should be but I removed the message because we could simply invalidate that cached value without restarting the tenant.

Reply:
Ok, I get it, now the value is cached in the `await _siteService.GetSiteSettingsAsync()` so it is invalidated when we update the value of the setting simply. So it is cached.